README.md

```diff
@@ -4,29 +4,29 @@ by zruncho
 
 ![Two filter halves mounted to an extrusion](Images/iso_black_4.png)
 
-### A recirculating air filter that fits in the tightest of spaces.
+### A recirculating air filter that fits in the tightest of spaces
 
 Mounts cleanly under the bed of an [F-Zero](https://github.com/zruncho3d/f-zero), [Micron](https://github.com/hartk1213/Micron), or other smaller-volume, "zero-size" enclosed printer.
 
-Heavily inspired by the [Nevermore series of air filters](https://github.com/0ndsk4/VoronUsers/tree/0ndsk4/printer_mods/0ndsk4/Nevermore_Air_Filter).
+Heavily inspired by the [Nevermore series of air filters](https://github.com/nevermore3d).
 
 Specs:
-* Uses standard 4010 blower fans, without modification
-* Prints easily, without supports, and no crazy overhangs
-* Lid attaches quickly and securely to the cartridge, for easy filter media changes
-* Cartridge snaps into place, magnetically, with a satisfying click
-* Includes a gasket slot for a perfect seal between base and cartidge
-* One fits in a 70mm x 97mm space; two fit in a 70mm x 194mm space
-* Mounts with a provided brace and one screw, or with high-bond tape
+
+- Uses standard 4010 blower fans, without modification
+- Prints easily, without supports, and no crazy overhangs
+- Lid attaches quickly and securely to the cartridge, for easy filter media changes
+- Cartridge snaps into place, magnetically, with a satisfying click
+- Includes a gasket slot for a perfect seal between base and cartidge
+- One fits in a 70mm x 97mm space; two fit in a 70mm x 194mm space
+- Mounts with a provided brace and one screw, or with high-bond tape
 
 See it in action here, in the serial request video for [F-Zero](https://github.com/zruncho3d/f-zero), a small flying-gantry printer; click the image to see the video:
 
 [![F-Zero serial request](https://img.youtube.com/vi/jLSPeJ7L9R4/0.jpg)](https://youtu.be/jLSPeJ7L9R4)
 
-
 ## Why add this to a printer?
 
-Go to the [Nevermore air filter](https://github.com/0ndsk4/VoronUsers/tree/0ndsk4/printer_mods/0ndsk4/Nevermore_Air_Filter) page to learn more about the value of filtering VOCs (Volatile Organic Compounds) in a printer.  A VOC filter should be required equipment for all printers.  
+Go to the [Nevermore air filter](https://github.com/nevermore3d/Nevermore_Micro) page to learn more about the value of filtering VOCs (Volatile Organic Compounds) in a printer. A VOC filter should be required equipment for all printers.
```

Comment:
best to deep-link to the current README that has all of the VOC info.  we'll publish this on a website on a later date.